### ISSUE TITLE
Convert HTML code to Next.js

### ISSUE THREAD

**Issue by Vinay-Khanagavi** [Vinay-Khanagavi/Cyberbear.github.io#5]
To enhance scalability and performance, I propose converting the current HTML-based code to a Next.js project. This will improve development efficiency, enable server-side rendering, and provide better structure for future contributions.

**Benefits:**

- Improved performance with server-side rendering
- Modular components for easier maintenance
- Enhanced developer experience with Next.js features

I’d appreciate any feedback or suggestions on this transition!

**Comment by biswa2005** [comment]
i want to do this issue please assign me but it will take some time

**Comment by ReihaneChamanifard** [comment]
I will be happy to help you in this project.

**Comment by B1N4RY-P4R45173** [comment]
### Hello @biswa2005 

Just wanted to confirm if you are still working on this?

**Comment by biswa2005** [comment]
No please unassigned me

**Comment by B1N4RY-P4R45173** [comment]
@ReihaneChamanifard Would like to continue with this?

You can join our discord server for any assistance that you may need with this https://discord.gg/YFWmkphf

**Comment by B1N4RY-P4R45173** [comment]
Hello @ReihaneChamanifard Can you please confirm if you are working on this?

**Comment by ReihaneChamanifard** [comment]
hello
Yes, of course I would like to cooperate
 I took a look at the codes and will join the discord server to talk more and divide the work.

**Comment by n-keerthi-gayathri** [comment]
@Vinay-Khanagavi is this issue available? can i work on it?

**Comment by B1N4RY-P4R45173** [comment]
I think @ReihaneChamanifard is already working on this. You can join our discord server and work with me. Or if u think u can submit a pr on your own u are free to do it aswell.

**Comment by ReihaneChamanifard** [comment]
Yes, I worked a little on it and I would like to work on it with you, I asked a question on a server about how we should proceed and what we should do, but I did not receive an answer. If we divide and prioritize the tasks, we can start. If possible, guide me in this matter.
